api/sip/__init__.py

```diff
@@ -238,7 +238,7 @@ def patron_information(self, username, password):
         except OSError as e:
             raise RemoteIntegrationException(self.server or "unknown server", str(e))
 
-    def _remote_patron_lookup(self, patron_or_patrondata):
+    def _remote_patron_lookup(self, patron_or_patrondata) -> Union[None, PatronData]:
```

Comment:
`Union[None, PatronData]` is isomorphic to `Optional[PatronData]`. I think `Optional` is clearer.

Reply:
Updated in most recent commit